src/csvtools.cpp

```diff
@@ -20,13 +20,25 @@ QString csvTools::assemble(QString call,
                            QString ctss,
                            QString ctsr)
 {
+    // handle submode
+    QString modeWithoutSubmode = QString("");
+    QString submode = QString("");
+    if (mode.contains(" / ")) {
+        QStringList modeParts = mode.split(" / ");
+        if (modeParts.size() == 2) {
+            modeWithoutSubmode = modeParts.at(0);
+            submode = modeParts.at(1);
+        }
+    }
+
     return QString("") +
            date + ", " +
            time + ", " +
            call + ", " +
            name + ", " +
            freq + ", " +
-           mode + ", " +
+           modeWithoutSubmode + ", " +
```

Comment:
Can you please use the name mode her? I think mode and submode would be enough :-), modeWithoutSubmode is a bit long IMHO

Reply:
@myzinsky Should I still change that? Just saw that you merged it anyway. Personally I prefer clear (even if long) names. As `mode` in the rest of the code means "mode + submode" I chose `modeWithoutSubmode` here.

Reply:
I will refactor it maybe after some time. 

Are you in Matrix? then you could join our discussion Chat there ...

Reply:
@myzinsky Yes I have the chat.darc.de account and just joined the discussion there